src/gluonts/dataset/jsonl.py

```diff
@@ -17,7 +17,8 @@
 from typing import NamedTuple
 
 # Third-party imports
-import ujson as json
```

Comment:
This also.

Reply:
Why do we want to remove this? If I remember correctly the data with `ujson` was much faster than with the stdlib `json` module.

Reply:
Sorry, what I meant is that I want to use it again, as it was replaced here with normal `json`.

I used a dataset for testing, which encoded nan-values as `NaN` instead of `"NaN"`, which caused ujson to be throw errors.

Reply:
> I used a dataset for testing, which encoded nan-values as `NaN` instead of `"NaN"`, which caused ujson to be throw errors.

I think I hit that issue at once in the past as well.